### PR TITLE
Added miner configuration parameters explanations.

### PR DESCRIPTION
**Pull request by jennijuju**

### CODE REVIEW

**Review by mishmosh — APPROVED** [review]
Can you separate the lotusminer/lotusstorage correct from the config params? I want to approve the former, but not the latter (needs more context about config.toml and how it works).

**Comment by hsanjuan** [comment]
Can we leave this unmerged and just include it in the new docs?

File: docs/mine/storage-mining.md [hunk 1]
```diff
@@ -122,3 +122,107 @@ Example:
 ```
 $ lotus-miner actor set-addrs /ip4/123.123.73.123/tcp/12345 /ip4/223.223.83.223/tcp/23456
 ```
+
+
+## Configuration
```
Comment (hsanjuan) [inline docs/mine/storage-mining.md:127]: Can this be PRed in `feat/docs-reorg`? There should be a placeholder guide (currently empty) called `miner-configuration.md`.

Comment (hsanjuan) [reply docs/mine/storage-mining.md:127]: Taken care of.

**Comment by johnnymatthews** [comment]
Looks like this stuff was manually ported over to the docs in the `feat/docs-reorg` branch. Closing.

**Pull request closed without merging**